br/use-br-command-line-tool: supplement br note (#6000)
pingcap/docs: br/use-br-command-line-tool.md

In the above command, `--db` specifies the name of the database to be restored. For descriptions of other options, see [Restore all backup data](#restore-all-the-backup-data)).

> > **Note**:

Reviewer: ```suggestion
> **Note:**
```

>

> > When you are restoring the backup data, the name of the database specified by `--db` must be the same as the one specified by `-- db` when the backup is in progress, otherwise the process fails. The reason is that the metafile `backupmeta` of the backup data records the database name so you can only restore data to the database with the same name. You are recommended to restore the backup data to the database with the same name of another cluster.

Reviewer: ```suggestion
> When you restore the backup data, the name of the database specified by `--db` must be the same as the one specified by `-- db` in the backup command. Otherwise, the restore fails. This is because the metafile of the backup data ( `backupmeta` file) records the database name, you can only restore data to the database with the same name. The recommended method is to restore the backup data to the database with the same name in another cluster.
```
